vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -178,31 +140,20 @@ Install the Angular elements package using `ng add` (not with `npm install` as i
 2. In the `<body>`, replace `<app-root></app-root>` with `<angular-widget />`:
 
    ``` html
-   <!doctype html>
-   <html lang="en">
-   <head>
-     <meta charset="utf-8">
-     <title>AngularWidget</title>
-     <base href="/">
-     <meta name="viewport" content="width=device-width, initial-scale=1">
-     <link rel="icon" type="image/x-icon" href="favicon.ico">
-   </head>
    <body>
      <angular-widget />
    </body>
-   </html>
    ```
-3. Go back to the bundle root directory
-4. Test run the Angular widget:
+3. Test run the Angular widget:
 ```sh
-ent bundle run angular-widget
+ng serve
```

Comment:
@nshaw why are we using angular command here instead of ent? is the previous ent command not working?

Reply:
@jyunmitch It dodges a node/ent compatibility situation you can run into with the build command. I hinted at that above with a new comment.